examples/reshare/src/application/scheme.rs

```diff
@@ -19,10 +19,10 @@ use std::{
 };
 
 /// The BLS12-381 threshold signing scheme used in simplex.
-pub type ThresholdScheme<V> = signing_scheme::bls12381_threshold::Scheme<V>;
+pub type ThresholdScheme<P, V> = signing_scheme::bls12381_threshold::Scheme<P, V>;
```

Comment:
This example could be simplified by hardcoding identity keys as ed25519 here.

Reply:
I like keeping it generic fwiw.